.github/workflows/release.yaml

```diff
@@ -12,11 +12,14 @@ jobs:
     runs-on: ubuntu-18.04 # 20.04 doesn't provide createrepo for now
     env:
       DOCKER_CLI_EXPERIMENTAL: "enabled"
+    permissions:
+      id-token: write
```

Comment:
I think we need more permissions since it disables other permissions.

>If you specify the access for any of these scopes, all of those that are not specified are set to none.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions

I'll take it.